tests/index.test.js

```diff
@@ -0,0 +1,66 @@
+const request = require("supertest");
+const app = require("../app.js");
+const fs = require("fs");
```

Comment:
You should not use `fs` at all unless you have a good reason. If you want to read / write to your DB, use the DB class you created